user/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+
+class User(models.Model):
+    userId         = models.CharField(max_length=50)
```

Comment:
```suggestion
    account         = models.CharField(max_length=50)
```
user_id이걸 할수 없다면 account라고 하는게 제일 나을것 같네요

Reply:
네! account로 수정했습니다~